app/app/public-cloud/requests/(request)/[id]/layout.tsx

```diff
@@ -13,9 +13,24 @@ import LightButton from '@/components/generic/button/LightButton';
 import Tabs, { ITab } from '@/components/generic/tabs/BasicTabs';
 import createClientPage from '@/core/client-page';
 import { comparePublicProductData } from '@/helpers/product';
+import { PublicCloudRequestGetPayload } from '@/queries/public-cloud-requests';
 import { getPublicCloudRequest } from '@/services/backend/public-cloud/requests';
 import { publicProductState } from '@/states/global';
 
+const mapToComponentData = (payload?: PublicCloudRequestGetPayload, isTest?: boolean) => {
```

Comment:
hm.. not sure why we need this mapper as `decision data` should have the flag `isTest`.

Reply:
public cloud product doesn't have `isTest` flag and this feature is not relevant for public cloud products